src/nwb_conversion_tools/tools/roiextractors/roiextractors.py

```diff
@@ -72,6 +77,55 @@ def get_default_ophys_metadata():
     return metadata
 
 
+def get_nwb_imaging_metadata(imgextractor: ImagingExtractor):
+    """
+    Convert metadata from the ImagingExtractor into nwb specific metadata.
+
+    Parameters
+    ----------
+    imgextractor: ImagingExtractor
+    """
+    metadata = get_default_ophys_metadata()
+    # Optical Channel name:
+    channel_name_list = imgextractor.get_channel_names()
+    if channel_name_list is None:
+        channel_name_list = ["generic_name"] * imgextractor.get_num_channels()
+
+    for index, channel_name in enumerate(channel_name_list):
+        if index == 0:
+            metadata["Ophys"]["ImagingPlane"][0]["optical_channel"][index]["name"] = channel_name
+        else:
+            metadata["Ophys"]["ImagingPlane"][0]["optical_channel"].append(
+                dict(
+                    name=channel_name,
+                    emission_lambda=np.nan,
+                    description=f"{channel_name} description",
+                )
+            )
+    # set imaging plane rate:
+    rate = (
+        np.float("NaN")
```

Comment:
I show `np.float("NaN")` as being deprecated soon - can you use `np.nan` instead?